README.md

```diff
@@ -14,6 +14,78 @@ By default there is little/no output to stdout or stderr until the script has fi
 
 When using `jellyfin.py`, the results can be saved to `json` using the `-j` parameter. These will be saved in a sub-directory in `pwd`. Saving the results as json also allows them to be checked in subsequent runs to skip already processed files.
 
+## Running in Docker
+  ### Parameters
+
+|| Parameter  | Function |
+| ---                                        | ---                                        | ---       |
+| Required | ```-e JELLYFIN_URL=http://Jellyfin:port``` | Jellyfin URL         |
+| Required | ```-e JELLYFIN_USERNAME=username```        | Jellyfin User Username        |
+| Required | ```-e JELLYFIN_PASSWORD='password'```      | Jellyfin User Password         |
+| Required | ```-v /path/to/config:/app/config```      | Location of config/data on disk. Must use the same locations for Jellyfin-Intro-Scanner & Jellyfin-Intro-Skipper containers to work correctly together.       |
+| Required | ```-v /path/to/media/on/host:/path/to/media/on/Jellyfin/container'```      |  Location of media library on disk. If you use the same volume path for your Jellyfin container, you don't have to edit ```path_map.txt``` in your config folder. (If you need to change it you must first create a ```path_map.txt``` in your config folder. ***Not in the data subfolder***)        |
+| Optional | ```-e CONFIG_DIR=/config```      |  Pointless to change the data directory in the docker container, but it's there if needed. If you do decide to add this make sure to update your ```/app/config``` path mapping.          |
+| Optional | ```-e DATA_DIR=/config/data'```      | Pointless to change the data directory in the docker container, but it's there if needed. If you do decide to add this make sure to update your ```/app/config``` path mapping.         |
```

Comment:
You're right that there isn't much use for adjusting these in the context of docker.

```suggestion
| Optional | ```-e DATA_DIR=/data'```      | Use a different directory to store cached data. Modifying this will likely require a new path mapping such as ```-v /path/to/data:/data```.         |
```